src/App.tsx

```diff
@@ -32,17 +38,16 @@ function App() {
         { path: "rooms", element: <Rooms /> },
         { path: "ads", element: <Ads /> },
         { path: "booking", element: <Bookings /> },
-        
-       
+        { path: "new", element: <AddNewAds /> },
```

Comment:
This path needs to be descended from the ads page, `ads/add-new-add` to indicate that we reach the add new ad page through the ads page.